foundation/oidc/src/main/kotlin/com/pingidentity/oidc/OidcClient.kt

```diff
@@ -139,24 +166,29 @@ open class OidcClient(private val config: OidcClientConfig) {
      *
      * @param token The access token to revoke. If null, the currently stored token is revoked.
      */
-    private suspend fun revoke(token: Token? = null) =
+    private suspend fun revoke(token: Token? = null) {
```

Comment:
we might want to have a public revoke method. Also allow to force using the AT or the RT on this